developer_document.md

```diff
@@ -499,9 +492,22 @@ index 894be0134d..a9a57c0a9e 100644
  {
    m.doc() = "cpp model python binding";
 ```
+# 3.	Accuracy evaluation
+## 3.1 Evaluate llm model in neural speed by lm_eval
+We can use this python script for accuracy evaluation.
+```
+python scripts/cal_acc.py --model hf_model --tasks lambada_openai,boolq,piqa,winogrande,hellaswag --use_autoround
+```
+
+Here is lm-evaluate result for [Intel/neural-chat-7b-v3-3-int4-inc](https://huggingface.co/Intel/neural-chat-7b-v3-3-int4-inc).
+Install lm_eval from pip version 0.4.2.
```

Comment:
better to also bump the `NS` version or commit-id

Reply:
done